### PR TITLE
Delete user

### PR DESCRIPTION
**Pull request by spiftire**
Tested. should work

### CODE REVIEW

File: server/src/main/java/no/ntnu/klubbhuset/service/UserService.java [hunk 1]
```diff
@@ -136,10 +137,14 @@ public Response createNewUser(String firstname, String lastname, String email, S
...
 
-        if ( user == null ) {
+        if (user == null) {
             return Response.status(Response.Status.NOT_FOUND).entity("User not found").build();
         }
 
+        TypedQuery<Member> query = entityManager.createQuery("delete from Member m where m.user = :user", Member.class);
+        query.setParameter("user", user);
+        query.executeUpdate();
+
```
Comment (0us) [inline server/src/main/java/no/ntnu/klubbhuset/service/UserService.java]: E ikkje det ryddigare å bruke cascading delete her?

Comment (0us) [inline server/src/main/java/no/ntnu/klubbhuset/service/UserService.java]: E ikkje det ryddigare å bruke cascading delete her?

File: server/src/main/java/no/ntnu/klubbhuset/domain/Order.java [hunk 2]
```diff
@@ -4,9 +4,11 @@
 
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 @Entity
 @Data
+@Table(name = "VIPPSORDERS")
```
Comment (markusranda) [inline server/src/main/java/no/ntnu/klubbhuset/domain/Order.java]: Should this really be commited in this pull request?

File: server/src/main/java/no/ntnu/klubbhuset/service/UserService.java (hunk 1 above)
Comment (markusranda) [reply server/src/main/java/no/ntnu/klubbhuset/service/UserService.java]: Det vil eg også ha

Comment (spiftire) [reply server/src/main/java/no/ntnu/klubbhuset/service/UserService.java]: Nope. [https://thoughts-on-java.org/avoid-cascadetype-delete-many-assocations/](url)

File: server/src/main/java/no/ntnu/klubbhuset/domain/Order.java (hunk 2 above)
Comment (spiftire) [reply server/src/main/java/no/ntnu/klubbhuset/domain/Order.java]: Nope. Den har lurt seg med, men har ikke så mye å si

**Pull request merged**